docs/HowToUseMultiHeap.md

```diff
@@ -0,0 +1,13 @@
+# How to use(add) Multi-heap Region
+
+Modify *[Kconfig](#Kconfig)* and *[TinyAra configurationfile](#TinyAra-configuration-file)*.  
+
+## Kconfig
+
+Add *select ARCH_HAVE_HEAPx* belong to config ARCH_ARM in os/arch/Kconfig.chip  
```

Comment:
File is wrong, not in os/arch/Kconfig.chip. We should add it in os/arch/<ARCH_NAME>/src/<CHIP_NAME>/Kconfig like below.
```
config ARCH_CHIP_<CHIP_NAME>
	select ARCH_HAVE_HEAPx
```